jina/clients/__init__.py

```diff
@@ -167,6 +170,39 @@ def delete(
             self._get_results, inputs, on_done, on_error, on_always, **kwargs
         )
 
+    def dump(
+        self,
+        path: str,
+        on_done: CallbackFnType = None,
+        on_error: CallbackFnType = None,
+        on_always: CallbackFnType = None,
+        shards: int = None,
+        formats: List[DumpTypes] = None,
+        **kwargs,
+    ) -> None:
+        """Issue 'update' request to the Flow.
+        :param path: path to the dump
+        :param formats: the formats into which to dump
+        :param shards: nr of shards in the dump
+        :param on_done: the function to be called when the :class:`Request` object is resolved.
+        :param on_error: the function to be called when the :class:`Request` object is rejected.
+        :param on_always: the function to be called when the :class:`Request` object is  is either resolved or rejected.
+        :param kwargs: additional parameters
+        :return: None
+        """
+        # TODO make Dump a control request to be passed to the Pod directly
```

Comment:
That's exactly the point. I think this function shouldn't be here anymore, since the control request is send via the `Flow`, non? Or are these control requests still send via the client? @florian-hoenicke